### PR TITLE
Chapter1/homework/5

### PR DESCRIPTION
**Pull request by mkasarci**
First Pull Request hope it's okay.

### CODE REVIEW

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 1]
```diff
@@ -9,7 +11,24 @@ public static class ArrayOperations
         /// <param name="array">Input array in a random order.</param>
         public static void Sort(int[] array)
         {
-            // ToDo: implement.
+            if (array != null)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs]: You can denest it:
if(array == null) return;

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 2]
```diff
@@ -9,7 +11,24 @@ public static class ArrayOperations
...
+            {
+                int temp;
+
+                for (int i = 0; i < array.Length - 1; i++)
+                {
+                    for (int j = (i + 1); j < array.Length; j++)
+                    {
+                        if (array[i] > array[j])
+                        {
+                            temp = array[i];
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs]: Can be extracted to swap fucntion

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 3]
```diff
@@ -63,8 +129,15 @@ public static int[] RemoveAt(int[] array, int index)
         /// <returns>A new array with element added at a given index. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertFirst(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:111]: Reuse the InsertAt function

File: Src/BootCamp.Chapter1/ArrayOperations.cs [hunk 4]
```diff
@@ -75,8 +148,16 @@ public static int[] InsertFirst(int[] array, int number)
         /// <returns>A new array with element added in the end of array. If an array is empty or null, returns new array with number in it.</returns>
         public static int[] InsertLast(int[] array, int number)
```
Comment (Almantask) [inline Src/BootCamp.Chapter1/ArrayOperations.cs:122]: Reuse InsertAt function

**Review by Almantask — CHANGES_REQUESTED** [review]
Be mindful of what you already have and where possible, reuse functions.

File: Src/BootCamp.Chapter1/ArrayOperations.cs (hunk 3 above)
Comment (mkasarci) [reply Src/BootCamp.Chapter1/ArrayOperations.cs:111]: Yeah I'm so silly

**Comment by mkasarci** [comment]
I've refactored my code. Can you check if it is okay?

**Review by Almantask — APPROVED** [review]
Sorry for missing this... No idea how did it slip my eyes for so long

**Pull request closed without merging**